### PR TITLE
Feature/networks initialization

### LINKED ISSUE #52: Add variables initializer to GaussianDynamicsModel
Consider applying some form of variable inititalizer to the dynamics models... maybe something along the lines of ...

```python
def normc_initializer(std=1.0):
    def _initializer(shape, dtype=None, partition_info=None):
        out = np.random.randn(*shape).astype(np.float32)
        out *= std / np.sqrt(np.square(out).sum(axis=0, keepdims=True))
        return tf.constant(out)

    return _initializer
```

### PR DESCRIPTION
**Pull request by 0xangelo**
Add kernel initialization to fully connected layers. 

Defaults taken from my [PyTorch implementations](https://github.com/angelolovatto/deep-rl/blob/master/proj/common/models.py), which in turn were based on [OpenAI Baselines](https://github.com/openai/baselines/blob/0182fe1877e95b2ef0a82747c20bed1523fb5a3f/baselines/a2c/utils.py#L58).

Closes #52

### CODE REVIEW

**Comment by thiagopbueno** [comment]
It's not clear to me that `normc_initializer` will be absolutely helpful... so it would be better if we could use it or not based on some config flag...

**Comment by 0xangelo** [comment]
> It's not clear to me that `normc_initializer` will be absolutely helpful... so it would be better if we could use it or not based on some config flag...

I'll add it to the model config.

File: mapo/agents/mapo/mapo_policy.py [hunk 1]
```diff
@@ -33,16 +33,17 @@ def build_mapo_losses(policy, batch_tensors):
...
     policy.loss_stats["critic_loss"] = critic_loss
     policy.loss_stats["actor_loss"] = actor_loss
     policy.mapo_losses = AgentComponents(
         dynamics=dynamics_loss, critic=critic_loss, actor=actor_loss
     )
-    return (
-        (dynamics_loss if dynamics_loss is not None else 0) + critic_loss + actor_loss
-    )
+    mapo_loss = critic_loss + actor_loss
+    if not config["use_true_dynamics"]:
```
Comment (thiagopbueno) [inline mapo/agents/mapo/mapo_policy.py:44]: My inner self now feels much better about this. Thanks! lol

**Review by thiagopbueno — APPROVED** [review]

File: mapo/agents/mapo/mapo_policy.py (hunk 1 above)
Comment (0xangelo) [reply mapo/agents/mapo/mapo_policy.py:44]: Yeah, now that I had more time I noticed how ugly it was 😅

**Pull request merged**